src/Compute/Compute/AvailabilitySets/UpdateAzureAvailabilitySetCommand.cs

```diff
@@ -79,6 +79,11 @@ public override void ExecuteCmdlet()
                                                 : this.AvailabilitySet.ProximityPlacementGroup
                     };
 
+                    if (avSetParams.ProximityPlacementGroup != null && string.IsNullOrEmpty(avSetParams.ProximityPlacementGroup.Id))
+                    {
+                        avSetParams.ProximityPlacementGroup.Id = null;
```

Comment:
correct me if I was wrong, this line is to ensure the value of Id is null not white spaces, am I right?

Reply:
Yes.  It ensures to send a null value for ID, which removes it from VM/VMSS/AvSet.